File: api/src/main/java/com/mystore/manager/api/admin/payload/ChangePasswordPayload.java
package com.mystore.manager.api.admin.payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.validation.constraints.NotBlank;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ChangePasswordPayload {
    @NotBlank
    private String oldPassword;
    @NotBlank
    private String newPassword;
    @NotBlank
    private String newPasswordConfirmed;
}

File: ui-admin/src/app/backend/payloads/admin/changepasswordpayload.ts

export class ChangePasswordPayload{
    oldPassword: string | null;
    newPassword: string | null;
    newPasswordConfirmed: string | null;
    
    constructor(){
        this.oldPassword = null;
        this.newPassword = null;
        this.newPasswordConfirmed = null;
    }
}